src/main/java/seedu/address/model/person/Person.java

```diff
@@ -21,16 +23,19 @@
     private final Email email;
 
     // Data fields
+    private final TransactionList<Debt> debts = new TransactionList<>();
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Set<Tag> tags) {
+    public Person(Name name, Phone phone, Email email,
```

Comment:
Maybe can have a separate constructor if the person doesn't want to enter debt? So kinda like how tags are optional, can make it such that if no debt entered then the debt is equal zero

Reply:
You mean when the user want to add a new person to the address book? When the user is adding a new person to the address book, they are not allowed to add debt. They can only do so through "people owe..." command. Also, this will be handled by PeopleAddCommandParser. PeopleAddCommandParser will create a person with empty debt. So actually, I didnt make debt optional. The user can't add debt when adding a new person.